src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -8,6 +8,16 @@
 
 namespace System
 {
+   public partial class AppDomain
```

Comment:
@wesh where do you suggest? Krzysztof suggested a new contract (System.Runtime.Remoting) but I don't think we're going to create a new one.

---
In reply to: [76925026](https://github.com/dotnet/corefx/pull/11275#discussion_r76925026) [](ancestors = 76925026)